test/bad_dtls_test.c

```diff
@@ -182,7 +185,7 @@ static int validate_client_hello(BIO *wbio)
     /* Update handshake MAC for second ClientHello (with cookie) */
     if (cookie_found && !EVP_DigestUpdate(handshake_md, data + MAC_OFFSET,
                                           len - MAC_OFFSET))
-            printf("EVP_DigestUpdate() failed\n");
```

Comment:
As long as it was a conscious decision and you think we're not losing any practical diagnostic info, I'm fine with it.